snudda/simulate/simulate.py

```diff
@@ -897,6 +901,10 @@ def add_synapse(self, cell_id_source, cell_id_dest, dend_compartment, section_id
 
         (channel_module, par_data) = self.synapse_parameters[synapse_type_id]
 
+        if channel_module is None:
```

Comment:
perhaps a warning would be in place here?

Reply:
Adding. Had it before, but there were so many printouts. Now added logic so that it will only print error once.